libs/ui/src/lib/metadata-card/metadata-card.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="cvi-metadata-card">
+  <div class="cvi-metadata-card__left-border"></div>
```

Comment:
It should be more concise to have the border applied to `__content` or the root element instead of a separate element?